app/assets/stylesheets/reservations/_index.css.scss

```diff
@@ -3,3 +3,7 @@
   border-radius: 5px;
   margin-bottom: 20px;
 }
+
+#reservations-list {
+  height: 10em;
```

Comment:
Didn't you teach me that the first step of code review is to read the relevant issue? Maybe I just made it up :stuck_out_tongue: